### PR TITLE
AC-591: Buttons should be Borderless

### PR DESCRIPTION
**Pull request by deepak140596**
## Description of what I changed

Made buttons in dialog box borderless.
Added style as buttonBarStyle to parents containing buttons
Added style as buttonBarButtonStyle to buttons inside the parent layout

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-591

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: openmrs-client/src/main/res/drawable/button_dialog_layout_background.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ripple xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (f4ww4z) [inline openmrs-client/src/main/res/drawable/button_dialog_layout_background.xml]: @deepak140596 Why is this file not in the `v21` folder?

**Comment by f4ww4z** [comment]
I think that after the button issue is resolved, we can rerun `gradlew lint` and check if this issue still exists somewhere in the code.

File: openmrs-client/src/main/res/drawable/button_dialog_layout_background.xml (hunk 1 above)
Comment (deepak140596) [reply openmrs-client/src/main/res/drawable/button_dialog_layout_background.xml]: Sorry for the late reply @f4ww4z , university exams took much of my time.
I forgot to add it to the v21 folder. I will do so tomorrow.

**Comment by deepak140596** [comment]
Closing this PR. Will work on AC-592.

**Pull request closed without merging**